<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindows</class>
 <widget class="QWidget" name="UpdateWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>958</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutprepare">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QComboBox" name="comboBox_Envt"/>
       </item>
       <item>
        <widget class="QLabel" name="lab_envt">
         <property name="text">
          <string>Environment</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutrelease">
       <item>
        <widget class="QTableView" name="tableView_envt"/>
       </item>
       <item>
        <widget class="QTableView" name="tableView_category"/>
       </item>
       <item>
        <widget class="QTableView" name="tableView_listcategory"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutcommands">
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_quit">
         <property name="text">
          <string>Abort</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
